Find a functionally equivalent code:android.graphics.Picture.beginRecording:COMMENT
Method Modifier: public      
Comment:/**
 * To record a picture, call beginRecording() and then draw into the Canvas
 * that is returned. Nothing we appear on screen, but all of the draw
 * commands (e.g. {@link Canvas#drawRect(Rect, Paint)}) will be recorded.
 * To stop recording, call endRecording(). After endRecording() the Canvas
 * that was returned must no longer be used, and nothing should be drawn
 * into it.
 */

@@ -1,5 +1,5 @@
 {
-    int ni = nativeBeginRecording(mNativePicture, width, height);
+    long ni = nativeBeginRecording(mNativePicture, width, height);
     mRecordingCanvas = new RecordingCanvas(this, ni);
     return mRecordingCanvas;
 }

